src/ConfigParser.cc

```diff
@@ -561,6 +562,20 @@ ConfigParser::rejectDuplicateDirective()
     throw TextException("duplicate configuration directive", Here());
 }
 
+SBuf
+ConfigParser::openDirective(const SBuf &line)
+{
+    Parser::Tokenizer tk(line);
+    SBuf directiveName; // TODO: Upgrade cfg_directive to SBuf and set it here.
```

Comment:
IMO `cfg_directive` should be removed in favour of a `ConfigParser` member, not just type changed.

Reply:
This change request is a duplicate of https://github.com/squid-cache/squid/pull/1840#discussion_r1649152617